### PR TITLE
Backend Basics: Demo

### PR DESCRIPTION
**Pull request by djfarly**
### 🔎 Review

Review via [🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/backend-basics_demo/sessions/backend-basics/demo-start?file=/README.md) or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/backend-basics_demo/sessions/backend-basics/demo-start -i
```

### ✨ Copyable Markdown for Curriculum

````md
Use this demo on
[🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/main/sessions/backend-basics/demo-start?file=/README.md)
or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/main/sessions/backend-basics/demo-start -i
```
````

### 🔎 Review

Review via [🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/backend-basics_demo/sessions/backend-basics/demo-end?file=/README.md) or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/backend-basics_demo/sessions/backend-basics/demo-end -i
```

### ✨ Copyable Markdown for Curriculum

````md
You can check out the final version of this demo on
[🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/main/sessions/backend-basics/demo-end?file=/README.md)
or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/main/sessions/backend-basics/demo-end -i
```
````

### CODE REVIEW

**Review by PReichetanz — COMMENTED** [review]
It works fine when used locally on my machine except `npm run dev`: the `--watch` flag is not accepted.

 CSB has some issues: 
- `npm run start` starts the server, but opening `http://127.0.0.1:8000/` results in "unable to connect at 127.0.0.1:8000/".
- `npm run dev` with `--watch` is not running here as well.

**Comment by PReichetanz** [comment]
> CSB has some issues:
> 
>     * `npm run start` starts the server, but opening `http://127.0.0.1:8000/` results in "unable to connect at 127.0.0.1:8000/".
> 
>     * `npm run dev` with `--watch` is not running here as well.

After upgrading to node18, it works fine.

File: sessions/backend-basics/demo-end/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "backend-basics_demo-end",
+  "version": "0.0.0-unreleased",
+  "description": "Backend Basics: Demo End",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "dev": "node --watch index.js",
```
Comment (djfarly) [inline sessions/backend-basics/demo-end/package.json:8]:
```suggestion
    "dev": "node --watch index.js || npm run start",
```

File: sessions/backend-basics/demo-start/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "backend-basics_demo-start",
+  "version": "0.0.0-unreleased",
+  "description": "Backend Basics: Demo Start",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "dev": "node --watch index.js",
```
Comment (djfarly) [inline sessions/backend-basics/demo-start/package.json:8]:
```suggestion
    "dev": "node --watch index.js || npm run start",
```

**Pull request merged**